projects/plugins/jetpack/tools/webpack.config.search.js

```diff
@@ -33,7 +33,7 @@ const baseWebpackConfig = getBaseWebpackConfig(
 				require.resolve( 'abortcontroller-polyfill/dist/polyfill-patch-fetch' ),
 			],
 		},
-		'output-chunk-filename': 'jp-search.chunk-[name]-[hash].min.js',
+		'output-chunk-filename': 'jp-search.chunk-[name].min.js?ver=[hash]',
```

Comment:
Does Webpack's configuration file allow for query string values like this?

Reply:
Yes it supports so, but is not documented. I guess it's because it's not the recommended way. 
Here's [what I found](https://github.com/webpack/webpack/issues/2329), a author of Webpack confirmed this approach. I don't see any resource indicating this would be deprecated, so I think we're good. But if it does get deprecated, we would need to implement our own plugin or loader I guess.

Reply:
PS: the file name would just be `jp-search.chunk-[name].min.js` while loading as `https://xxxx/path/jp-search.chunk-[name].min.js?ver=hash`

Reply:
What do you think of the failure scenario the author penned [here](https://github.com/webpack/webpack/issues/2329#issuecomment-210433116)? Do you think it could cause failures in any of our environments (e.g. simple, atomic, and Jetpack)?

Reply:
Yep we talked about this before (pcNPJE-9r-p2), query based cache buster could cause inconsistency. But it wouldn't in our case.

- It only would happen when user has loaded one chunk, and we updated assets, and user loads another chunk and unluckily enough, the chunk would break the app. In our case, this will NOT happen because we have ONLY one lazy-loaded chunk, at least for now.
- Even it happens, it's recoverable - a refresh would do.
- Hashed file name has its problem too, like what happened [here](https://github.com/Automattic/jetpack/issues/19752)

So I don't think the approach would cause issues for Simple, Atomic or Jetpack sites theoretically. But of course we'll need to watch the environments carefully after delivering the changes.

Reply:
For wpcom it seems there are more than one chunks, but the conclusion still stands I guess.